stop-mycroft.sh

```diff
@@ -90,30 +91,28 @@ OPT=$1
 shift
```

Comment:
It seems that `sh` doesn't handle shifting when there are no arguments.
```
./mycroft-core/stop-mycroft.sh: 91: shift: can't shift that many
```

So perhaps we need a conditional on here?

Reply:
That is strange, that shouldn't be the case. Afaik if none argument is specified, it should just do `N+1`.
Shellcheck doesn't complain either. How did you get this error? I can't reproduce it personally.

I could do `shift 1` but that seems redundant. What conditional would you suggest?

Reply:
Apparently the expected behaviour in [ash](https://linux.die.net/man/1/ash):
> Shift the positional parameters n times. A shift sets the value of $1 to the value of $2, the value of $2 to the value of $3, and so on, decreasing the value of $# by one. If n is greater than the number of positional parameters, shift will issue an error message, and exit with return status 2.

What about:
```
OPT=$1
if [ $# -gt 0 ] ; then
    shift
fi
```

Reply:
Yup thanks for that pointer. My system doesn't run Ash (`/bin/sh` points to Busybox) for me so I didn't catch this.

Modified it, could you try again?